test/test-logger.js

```diff
@@ -27,8 +33,218 @@ console._stdout.write = function() {
   orig.apply(this, arguments);
 };
 
+describe('mocking fd logger with config', function () {
+  it('should write an entry to the fs mock when log is called', function () {
+    var logger = require('../lib/logger.js');
+    var l = logger.create(logger.SILLY, PREFIX, 9);
+    for (var i = 0; i < 15; i += 1) {
+      l.info('info_');
+    }
+    var s = fs.readFileSync(MOCK_PATH).toString();
+    assert.deepEqual(s.split('\n').length, 10, 'There should be 10 entries in the logs')
+  });
+  before(function () {
+    var mockTree = {};
+    mockTree[os.tmpdir()] = {};
+    mockTree[os.tmpdir()][MOCK_FILE] = '';
+    fsMock(mockTree, {createTmp: false});
+  });
+  after(function () {
+    fsMock.restore();
```

Comment:
Does this clean up tmp files or only remove the file redirects introduced by the mocks?